gnd/src/main/java/com/google/android/gnd/model/basemap/tile/TileSource.java

```diff
@@ -22,6 +22,11 @@
 @AutoValue
 public abstract class TileSource {
 
+  /** Increment the area reference count of a tile source by one. */
+  public static TileSource incrementAreaCount(TileSource tileSource) {
```

Comment:
Since this operates on a tile source, can you make this non-static and modify `this` instead?

Reply:
Done, thanks!